container/shim/src/index.js

```diff
@@ -82,15 +79,7 @@ const handleCID = asyncHandler(async (req, res) => {
     return;
   }
 
-  const isBifrostGateway = req.headers["user-agent"]?.includes("bifrost-gateway");
-  const isSampled = Math.random() < LASSIE_SAMPLE_RATE;
-  const useLassie = isBifrostGateway || isSampled;
-
-  if (useLassie && LASSIE_ORIGIN) {
-    return respondFromLassie(req, res, { cidObj, format });
-  }
-
-  respondFromIPFSGateway(req, res, { cid, format });
+  return respondFromLassie(req, res, { cidObj, format });
```

Comment:
```suggestion
  respondFromLassie(req, res, { cidObj, format });
```

Reply:
Given we are using the asyncHandler wrapper, don't we want to capture async errors from the `respondFromLassie` function?

Reply:
good point!